biogeochem/EDCanopyStructureMod.F90

```diff
@@ -1616,12 +1625,10 @@ subroutine leaf_area_profile( currentSite )
                 ! is obscured by snow.
 
                 layer_top_height = currentCohort%height - &
-                     ( real(iv-1,r8)/currentCohort%NV * currentCohort%height *  &
-                     prt_params%crown_depth_frac(currentCohort%pft) )
+                     ( real(iv-1,r8)/currentCohort%NV * crown_depth )
```

Comment:
nice update here @mpaiao 

This will  have a conflict with the new two-stream PR.  I created a new subroutine that calculates the leaf area of crown layers, and it will need to leverage your fixes, see: https://github.com/NGEET/fates/pull/1141/files#diff-4fd8508e4a25680b45ecbfec6b6d80980a5b1e358af731203ee9d29266be46dbR2655

Reply:
Thanks for spotting this, @rgknox. Once the two-stream PR is integrated, I can update this part. I will leave this conversation open so we don't forget.

Reply:
@rgknox Should I go ahead and edit subroutine `VegAreaLayer` now that two-stream has been merged?